### DISCUSSION TITLE
v2.8.1

### DISCUSSION THREAD

**Discussion by NayamAmarshe** [upscayl/upscayl#467] [Releases]
**Upscayl v2.8.1 Release Discussions**

Comment down below if you have anything specific to discuss about v2.8.1

**Comment by VinzMota** [comment]
Hi 

I'm having worst results using the "digital art" model since I updated

![test1](https://github.com/upscayl/upscayl/assets/145080602/ec1c3c51-ed26-4f63-a56a-426d9438de57)


![test2](https://github.com/upscayl/upscayl/assets/145080602/4cc5942e-579a-4baf-ba5f-11b701dc6178)

**Reply by NayamAmarshe** [reply]
I suppose it has something to do with the compression. Try PNG output at 4x scale and set the compression to 0.

**Reply by VinzMota** [reply]
Those are the settings I used. I went back to 2.7.5 and my results are good again

**Reply by aaronliu0130** [reply]
I don't really get the problem, may somebody elaborate?

**Reply by VinzMota** [reply]
before 2.8.1, "digital art" would produce smooth colors and gradients , after 2.81, it looks like colors are pixelated

**Reply by VinzMota** [reply]
2.7.5
![2 7 5](https://github.com/upscayl/upscayl/assets/145080602/340a45eb-ac12-4aeb-a3bc-d155bac529fc)


2.8.1
![2 8 1](https://github.com/upscayl/upscayl/assets/145080602/fd835a9f-3004-4061-9c35-918bcc7dc616)

**Reply by aaronliu0130** [reply]
Thanks, this is now tracked in #474 .

**Reply by SpiritPowerTX** [reply]
Hello ! I have the same issue the same problem and it is boring.
I have found a weird solution : that when I launch a very big batch images folder upscaling the GPU turn hot (all is ok for now) BUT AFTER the GPU job finished the CPU start to turn hot for destroy the good finished pictures for replace the good ones by the ugly pixelated ones. So the solution is : when the job is finished (for the GPU) I imediatly kill the program process "Upscayl" while it is using the CPU and i have my nice pictures without the ugly pixels

**Comment by Beherit6666** [comment]
I keep receiving that error at the moment i update to 2.8.1
![Screenshot_3](https://github.com/upscayl/upscayl/assets/145681713/58c1ed31-7e0b-4ba0-b06c-1bd0d273f66e)

**Reply by aaronliu0130** [reply]
Please see #463 and keep your stuff to the same thread. Should also be fixed in 2.8.5

**Comment by Beherit6666** [comment]
I try to do that
![Screenshot_4](https://github.com/upscayl/upscayl/assets/145681713/fd8fd38f-8a36-4000-8895-13fc6c42dd66)

**Comment by Beherit6666** [comment]
![Screenshot_5](https://github.com/upscayl/upscayl/assets/145681713/c712f524-992c-49bc-a889-e833a965a4b4)
